.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 typings
 flow-typed
 coverage
+.idea
```

Comment:
it is not good idea to ignore tool specific folders in project gitignore
can you remove this change and I'll merge this PR?